ivy_tests/test_docstrings.py

```diff
@@ -285,6 +338,21 @@ def test_docstrings(backend):
         "dropout",
         "dropout1d",
         "dropout3",
+        # generate different results due to randomization
+        "svd",
+        # wrong examples/implementations
+        "conv2d_transpose",
```

Comment:
wrong examples should be fixed or removed. The tests should not be skipped for these.

Reply:
Just cleared those, at next PR I'll solve array method docs with ivy functional docs.